crates/pallets/starknet/src/message.rs

```diff
@@ -0,0 +1,91 @@
+use frame_support::BoundedVec;
+use kp_starknet::execution::{CallEntryPointWrapper, ContractAddressWrapper, EntryPointTypeWrapper};
+use kp_starknet::transaction::types::Transaction;
+use sp_core::{H256, U256};
+
+use crate::pallet::alloc::borrow::ToOwned;
+use crate::pallet::alloc::format;
+use crate::pallet::alloc::string::String;
+use crate::pallet::alloc::vec::Vec;
+use crate::types::Message;
+
+pub const LAST_FINALIZED_BLOCK_QUERY: &str =
+    r#"{"jsonrpc": "2.0", "method": "eth_getBlockByNumber", "params": ["finalized", true], "id": 0}"#;
+
+#[inline(always)]
+pub fn get_messages_events(from_block: u64, to_block: u64) -> String {
+    format!(
+        "{{
+            \"jsonrpc\": \"2.0\",
+        \"method\": \"eth_getLogs\",
+        \"params\": [
+            {{
+                \"fromBlock\": \"0x{:x}\",
+                \"toBlock\": \"0x{:x}\",
+                \"address\": \"0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4\",
```

Comment:
Hardcoded values ?

Reply:
Still no config + the starknet core contract will most likely not change

Reply:
Still we should put that in the runtime config imo. Upgrade process should be thought ahead of time